api/base.proto

```diff
@@ -227,3 +227,7 @@ message TransportSocket {
   // See the supported transport socket implementations for further documentation.
   google.protobuf.Struct config = 2;
 }
+
+message Percent {
```

Comment:
Can you add some simple comment for docs purposes (e.g. explaining purpose, valid ranges).

Reply:
Yep, fixed.